### PR TITLE
Show version information for Plans if available

### PR DESCRIPTION
**Pull request by shefalijoshi**
Closes [VIPEROMCT-51]

### Describe your changes:
Show plan version in formation if it is available

### All Submissions:

* [ ] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [ ] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [ ] Changes address original issue?
* [ ] Tests included and/or updated with changes?
* [ ] Command line build passes?
* [ ] Has this been smoke tested?
* [ ] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate unit tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Commit messages meet standards?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

**Comment by unlikelyzero** [comment]
@shefalijoshi let me know when you have 30 minutes to talk about a visual test for this one

File: src/ui/inspector/details/Properties.vue [hunk 1]
```diff
@@ -95,6 +95,7 @@ export default {
             const timestampLabel = this.domainObject.modified ? 'Modified' : 'Created';
             const timestamp = this.domainObject.modified ? this.domainObject.modified : this.domainObject.created;
             const notes = this.domainObject.notes;
+            const version = this.domainObject.version;
```
Comment (jvigliotta) [inline src/ui/inspector/details/Properties.vue:98]: This is nice, now any other objects that have versions will also display them (if we have any).

**Review by jvigliotta — APPROVED** [review]
This looks great, smoke tested locally, no issues.

**Comment by jvigliotta** [comment]
@shefalijoshi please create an issue for this (might as well close it and add the unverified label as well).

**Comment by unlikelyzero** [comment]
https://github.com/nasa/openmct/issues/5782

**Pull request merged**